pywbemtools/pywbemcli/_context_obj.py

```diff
@@ -276,6 +239,24 @@ def verbose(self):
         """
         return self._verbose
 
+    def pywbem_server_exists(self):
+        """
+        Return True if a pywbem_server is defined.  This method allows testing
+        for the existence of a PywbemServer object without causing an exception.
+        The normal method is to simply access the property. In that case,
+        an exception will be generated if there is no PywbemServer defined.
+
+        Returns:
+            :class": `boolean True if an instance of PywbemServer is defined.
+        """
+        return self._pywbem_server is not None
+
+    def set_connection(self, connection):
+        """ Set the connection parameter as the current connection object and
+            establish the new connection
+        """
+        self._pywbem_server = connection
```

Comment:
There is a pywbem server setter method. I don't think there should be an additional `set_connection()` method that does the same thing.

Reply:
The good news is that this is not even used. Removed.